src/netspresso_trainer/models/op/custom.py

```diff
@@ -489,3 +489,142 @@ def forward(self, x: Tensor) -> Tensor:
 
     # def __repr__(self):
     #     return "{}(type={})".format(self.__class__.__name__, self.pool_type)
+
+
+class Focus(nn.Module):
```

Comment:
Self-note:

This would be not compatible with NP Compressor because of reshaping including channel dim.

Reply:
I tested CSPDarknet with PyNP, and fortunately, it works well.